beacon-chain/core/electra/upgrade.go

```diff
@@ -15,6 +15,129 @@ import (
 	"github.com/pkg/errors"
 )
 
+// ConvertToElectra converts a Deneb beacon state to an Electra beacon state. It does not perform any fork logic. 
```

Comment:
This adds code that is repeated from `UpgradeToElectra`  without computing some of the missing fields because 
1. In the state-diff package there is no need to compute the actual values of the remaining fields. 
2. Wanted to avoid changing production code that could be run at runtime.